kitsune/users/validators.py

```diff
@@ -0,0 +1,6 @@
+from django.core.validators import RegexValidator
+
+
+TwitterValidator = RegexValidator(r'^[\w]+$',
```

Comment:
Hmm. Actually, I tested it, and I guess `\w` does match digits and underscores. It would still be good to add a test for it.